src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx

```diff
@@ -1,24 +1,26 @@
 ---
-title: Switch to the new pricing model
+title: Switch to the New Relic One pricing model
 metaDescription: "How to switch to New Relic's newer pricing model."
 redirects: 
   - /docs/accounts/accounts-billing/switch-new-model
 ---
 
-This doc explains how to switch from our original pricing model to the New Relic One pricing model. [Learn more about pricing models.](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models)
+Some of our older customers are still on our **original pricing model**. Our newer pricing model, called the **New Relic One pricing model**, is much simpler and offers many benefits compared to our original pricing model ([learn more](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing)). This doc explains the options available for switching to the newer pricing model. 
```

Comment:
Does original pricing model need any caps?

Reply:
no, it doesn't. it's meant to be a bit of an informal wording. might just seem strange cause it's capped.